ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionProperties.java

```diff
@@ -91,6 +123,9 @@ public IngestionProperties(String databaseName, String tableName)
         QueueAndTable
     }
 
+    /**
+     * Validate the minimum non-empty values needed for data ingestion.
+     */
     public void validate(){
         if(StringUtils.isAnyEmpty(databaseName,tableName) || reportMethod == null){
```

Comment:
Please separate these errors, also check if we should use empty or blank

Reply:
Changed to use the validation methods in the new helper